package.json

```diff
@@ -37,8 +37,8 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@blackbaud/skyux": "2.38.0",
+    "@blackbaud/skyux": "2.40.0",
     "@blackbaud/skyux-builder": "1.30.0",
```

Comment:
Since we're updating packages, Builder is up to `1.31.1` now.

Reply:
done